doc/release_notes/release_notes_2.0.rst

```diff
@@ -287,12 +287,43 @@ Many new and updated `reference documents <https://projectacrn.github.io>`_ are
   * :ref:`vtd-posted-interrupt`
 
 
-Fixed Issues
-************
 
-.. comment- :acrn-issue:`1773` - [APLNUC][IO][LaaG]USB Mediator USB3.0 and USB2.0 flash disk boot up UOS, quickly hot plug USB and Can not recognize all the devices
+Fixed Issues Details
+********************
+- :acrn-issue: `3715` -  Add support for multiple RDT resource allocation and fix L3 CAT config overwrite by L2
```

Comment:
delete the space, should be `` :acrn-issue:`3715` ``  (as described in the comment that was there)